doc/page01_info.md

```diff
@@ -94,55 +99,60 @@ cmake
 ```
 
 ## VCPKG
+
 **EVE** can be fetched from [VCPKG](https://vcpkgx.com/details.html?package=eve). Note that, as of
 now, we still don't support MSVC.
 
 ```bash
-$ vcpkg install eve
+vcpkg install eve
 ```
 
 # Installation from Source
+
 If you didn't fetched **EVE** from a package manager, you'll need to install it via our CMake
 system.
 
 ## Setting up the Library
-Create a `build` directory here and enter it. Once in the `build` directory, you can use  **CMake**
-to generate the build system for **EVE**.
 
-We recommend using [Ninja](https://ninja-build.org/) but any build system is fine.
+With **CMake**, generate the build system for **EVE**. We recommend using
+[Ninja](https://ninja-build.org/) but any build system is fine.
 
 @verbatim
-$ mkdir build
-$ cd build
-$ cmake .. -G Ninja
+cmake -B build -G Ninja
 @endverbatim
 
 Once **CMake** completes, you can use the `install` target to build and install **EVE**. By default,
-the library will be installed in the `/usr/local`directory, thus requiring root privileges on Linux.
+the library will be installed in the `/usr/local` directory, thus requiring root privileges on Linux.
 
-@verbatim
-$ ninja install
-@endverbatim
+```bash
+cmake --install build                # cmake >=3.15
+cmake --build build --target install # cmake <3.15
```

Comment:
I was following EVE's [top-level CMake](https://github.com/jfalcou/eve/blob/main/CMakeLists.txt#L6), which indicated 3.2.
We can definitely simplify the documentation to just using the *--install* command, but I would recommend at least requiring 3.15, or 3.19, in the top level to avoid misuse.

Reply:
Feel free to add it there. This 3.2 makes little sense.

Reply:
Since CMake 3.19 commands are used elsewhere, I'll ensure to update the minimum required CMake version to 3.19 as part of this PR.

Just to be transparent, the *--install* command won't invoke the *install* target but will instead run the installation rules generated by the CMake configure step. There's nothing to build with this header-only library, so an intermediate *--build* command isn't needed. Building the *install* target builds everything in *all*, too. I believe that *--install* is preferred because it doesn't override the default install prefix, CMAKE_INSTALL_PREFIX, that CMake deduced through its system introspection, and allows the prefix changes w/o re-configuring + re-building.

The reason I'm mentioning this is that I will also slightly simplify the wording of the above text to no longer mention the *install* target or the building, and wanted to run this by you before I make another push and burn more of your CI credits. I can post a diff or just make the push, too.

Reply:
Don't hesitate to fire anytime, the CI credit is not an issue

Reply:
Okay. I increased the minimum required CMake version to 3.19 and updated docs to reflect those newer commands.

Reply:
Guess I was wrong XD 3.18 it is. Sorry for the mishmash.

Reply:
changed to 3.18. Any changes you'd like or can this conversation be resolved?